.travis.yml

```diff
@@ -20,7 +22,7 @@ before_script:
 #  - php app/console doctrine:fixtures:load --env=test --no-interaction
 
   # install required gulp-packages
-  - npm install gulp@"^3.9.0" gulp-rename@"^1.2.2" gulp-coffee@"^2.3.1" gulp-util@"^3.0.6" -g
+  - npm install gulp@"^3.9.0" gulp-rename@"^1.2.2" gulp-coffee@"^2.3.1" gulp-util@"^3.0.6"
```

Comment:
Wouldn't it be beter to use yarn? See https://blog.travis-ci.com/2016-11-21-travis-ci-now-supports-yarn
With yarn, we can use our yarn.lock-file and make sure we have the correct dependencies installed.